--- host-build.mk.wrt	2013-03-21 14:49:00.000000000 +0100
+++ host-build.mk	2013-05-09 14:45:02.079042404 +0200
@@ -1,5 +1,6 @@
 #
 # Copyright (C) 2006-2010 OpenWrt.org
+# Copyright (C) 2013 Freetz.org
 #
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
@@ -9,17 +10,17 @@
 HOST_INSTALL_DIR ?= $(HOST_BUILD_DIR)/host-install
 HOST_BUILD_PARALLEL ?=
 
-ifneq ($(CONFIG_PKG_BUILD_USE_JOBSERVER),)
+ifneq ($(FREETZ_PKG_BUILD_USE_JOBSERVER),)
   HOST_MAKE_J:=$(if $(MAKE_JOBSERVER),$(MAKE_JOBSERVER) -j)
 else
-  HOST_MAKE_J:=-j$(CONFIG_PKG_BUILD_JOBS)
+  HOST_MAKE_J:=-j$(FREETZ_PKG_BUILD_JOBS)
 endif
 
 ifeq ($(strip $(HOST_BUILD_PARALLEL)),0)
 HOST_JOBS?=-j1
 else
-HOST_JOBS?=$(if $(HOST_BUILD_PARALLEL)$(CONFIG_PKG_DEFAULT_PARALLEL),\
-	$(if $(CONFIG_PKG_BUILD_PARALLEL),$(HOST_MAKE_J),-j1),-j1)
+HOST_JOBS?=$(if $(HOST_BUILD_PARALLEL)$(FREETZ_PKG_DEFAULT_PARALLEL),\
+	$(if $(FREETZ_PKG_BUILD_PARALLEL),$(HOST_MAKE_J),-j1),-j1)
 endif
 
 include $(INCLUDE_DIR)/host.mk
@@ -53,7 +54,7 @@
 HOST_CONFIGURE_VARS = \
 	CC="$(HOSTCC)" \
 	CFLAGS="$(HOST_CFLAGS)" \
-	CPPFLAGS="$(HOST_CPPFLAGS)" \
+	CPPFLAGS="$(HOST_CFLAGS)" \
 	LDFLAGS="$(HOST_LDFLAGS)" \
 	SHELL="$(BASH)"
 
@@ -71,10 +72,10 @@
 
 HOST_MAKE_FLAGS =
 
-HOST_CONFIGURE_CMD = $(BASH) ./configure
+HOST_CONFIGURE_CMD = ./configure
 
 ifneq ($(HOST_OS),Darwin)
-  ifeq ($(CONFIG_BUILD_STATIC_TOOLS),y)
+  ifeq ($(FREETZ_BUILD_STATIC_TOOLS),y)
     HOST_STATIC_LINKING = -static
   endif
 endif
@@ -129,7 +130,6 @@
   SUBDIR:=$(PKG_SOURCE_SUBDIR)
   VERSION:=$(PKG_SOURCE_VERSION)
   MD5SUM:=$(PKG_MD5SUM)
-  MIRROR_MD5SUM:=$(PKG_MIRROR_MD5SUM)
 endef
 
 define Host/Exports/Default
@@ -164,7 +164,6 @@
 	touch $$@
 
   $(call Host/Exports,$(HOST_STAMP_BUILT))
-  $(HOST_STAMP_BUILT): $(HOST_STAMP_CONFIGURED)
 		$(foreach hook,$(Hooks/HostCompile/Pre),$(call $(hook))$(sep))
 		$(call Host/Compile)
 		$(foreach hook,$(Hooks/HostCompile/Post),$(call $(hook))$(sep))
@@ -187,8 +186,8 @@
 
   host-prepare: $(HOST_STAMP_PREPARED)
   host-configure: $(HOST_STAMP_CONFIGURED)
-  host-compile: $(HOST_STAMP_BUILT) $(if $(STAMP_BUILT),$(HOST_STAMP_INSTALLED))
-  host-install: $(HOST_STAMP_INSTALLED)
+  host-compile: $(HOST_STAMP_BUILT)
+  host-install:
   host-clean: FORCE
 	$(call Host/Clean)
 	$(call Host/Uninstall)
